### DISCUSSION TITLE
multiple species search

### DISCUSSION THREAD

**Discussion by haiyan-MS** [vdemichev/DiaNN#917] [Q&A]
HI  We are trying to publish a paper that used DIA NN to analyze vaginal fluid of mice. Thus, mice and candida proteome realistically present in the sample. We generated predicted libary using Fasta fiile from both species. Now Would you help me with how to answer thsi reviewer question? Thanks. 
About proteome analysis. The same fluid samples were analyzed for the presence of both mouse and candida proteins. How many tryptic fragments were sufficient to identify the protein? Please add this information to supplement file. (I guess I can say we filtered it using FDR 1% for both protein and peptide at individual sample level)
Could the same tryptic fragments, when analyzed, be considered by the program as components of different proteins (mice and candida)? How was this information subtracted? This is where I need the most help. 

Thanks.

Haiyan

**Comment by vdemichev** [comment]
Hi Haiyan,

> How many tryptic fragments were sufficient to identify the protein? 

Protein false discovery rate in DIA-NN is controlled statistically, using a target-decoy model. That is, a protein can be identified with a single very high-confidence peptide ID or with two moderate-confidence peptide IDs, for example. This approach has been validated e.g. in https://www.nature.com/articles/s41467-022-31492-0, with a two-species method, wherein calls of decoy species proteins allow to estimate the true false discovery rate and show that it matched the protein group q-values reported by DIA-NN.

> Could the same tryptic fragments, when analyzed, be considered by the program as components of different proteins (mice and candida)?

Indeed, this could happen, but is an extremely rare occurance, due to low sequence identity between animals and unicellular fungi. In such cases the software assigns the peptides to whatever protein it thinks they are most likely originating from, based on a maximum parsimony principle (similar to IDPicker https://pubs.acs.org/doi/abs/10.1021/pr900360j). In case of mammalian-yeast proteomes this algorithm is reliable enough that the data is properly quantitative https://elifesciences.org/articles/83947. Nevertheless, any peptides that could originate from either of the species can be filtered out, if desired, using the Protein.Ids column in the main DIA-NN report and some R/Python package that can load the respective FASTAs and match the entries in Protein.Ids to the corresponding species.

Best,
Vadim

**Comment by haiyan-MS** [comment]
Dear Vadim:

Thank you so much for this detailed reply. This is way more than I expected. I am sure this is enough to convince the reviewer, although we as users have been convinced a long time ago.

Haiyan
________________________________
